internal/client/channel.go

```diff
@@ -0,0 +1,15 @@
+package client
+
+import (
+	"sync"
+
+	"github.com/gortc/turn"
```

Comment:
Can we pull `turn.ChannelNumber`, just so we don't have any dependencies we don't control!

Reply:
The type is actually associated with methods for message building/parsing. I'd like to the dependency removal after landing this TURN client's functional issue.

Reply:
ACK, sounds good to me!